stamina-testkit/src/test/scala/stamina/testkit/ScalatestTestGenerationSpec.scala

```diff
@@ -11,48 +11,71 @@ class ScalatestTestGenerationSpec extends StaminaTestKitSpec {
 
   import TestDomain._
 
-  case class ItemPersister(override val key: String) extends Persister[Item, V1](key) {
+  case class ItemPersister[V <: Version: VersionInfo](override val key: String) extends Persister[Item, V](key) {
```

Comment:
Gave it a go in https://github.com/scalapenos/stamina/pull/63 - what do you think?